bin/test/main.ml

```diff
@@ -230,7 +230,7 @@ let read_part file part =
         file
   | Some lines ->
       let contents = String.concat ~sep:"\n" lines in
-      String.trim contents
+      String.trim contents ~drop:(function '\n' -> true | _ -> false)
```

Comment:
Won't that leave trailing whitespaces when it used not to?
Could we do something like a `drop ~rev:true` followed by a `drop ~sat:(function '\n' -> true | _ false` to preserve that behaviour?

Reply:
It should be fixed now, I modified the test to check this.